validation-test/Serialization/SR7879.swift

```diff
@@ -0,0 +1,10 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -emit-module-path %t/main4.swiftmodule -swift-version 4 %s
+// RUN: %target-build-swift -emit-module-path %t/main4_2.swiftmodule -swift-version 4.2 %s
+
+// REQUIRES: OS=ios
+
+import UIKit
+
+_ = UIEdgeInsets.zero
```

Comment:
Can you try defining default arguments with these values as well so that we can test serializing references to these globals too?

Reply:
Good call. The original bug showed up just a mention of the SIL global in the partial swiftmodules, but having an actual reference to it is important.